x-pack/plugins/alerting/public/hooks/use_get_maintenance_window.test.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { renderHook } from '@testing-library/react-hooks/dom';
+import { waitFor } from '@testing-library/dom';
+
+import { AppMockRenderer, createAppMockRenderer } from '../lib/test_utils';
+import { useGetMaintenanceWindowsList } from './use_get_maintenance_windows_list';
+
+const mockAddDanger = jest.fn();
+
+jest.mock('../utils/kibana_react', () => {
+  const originalModule = jest.requireActual('../utils/kibana_react');
+  return {
+    ...originalModule,
+    useKibana: () => {
+      const { services } = originalModule.useKibana();
+      return {
+        services: {
+          ...services,
+          notifications: { toasts: { addDanger: mockAddDanger } },
+        },
+      };
+    },
+  };
+});
+jest.mock('../services/maintenance_windows_api/get', () => ({
+  getMaintenanceWindow: jest.fn(),
+}));
+
+const { getMaintenanceWindow } = jest.requireMock('../services/maintenance_windows_api/get');
+
+let appMockRenderer: AppMockRenderer;
+
+describe('useGetMaintenanceWindow', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+
+    appMockRenderer = createAppMockRenderer();
+  });
+
+  it('should call onError if api fails', async () => {
+    getMaintenanceWindow.mockRejectedValue('');
+
+    renderHook(() => useGetMaintenanceWindowsList(), {
```

Comment:
This is the wrong hook 🤦‍♀️

Reply:
Resolved in https://github.com/elastic/kibana/pull/154669/commits/c3dddf199a3800aad7fb385bcf178bbcf73334a8